src/css/Audition.css

```diff
@@ -1,76 +1,85 @@
+/* WEB DESIGNS */
 .font {
-    font-family: Arial;
+  font-family: Arial;
 }
-
 .hideHorizontalScroll {
-    overflow-x: hidden;
+  overflow-x: hidden;
 }
-
 .audition-info {
-    color: #FF4081;
+  color: #ff4081;
 }
-
 .title {
-    font-size: 110px;
+  padding-top: 7%;
+  font-size: 7em;
 }
-
 .audition-text {
-    padding-right: 7%;
-    padding-left: 5%;
+  padding-right: 7%;
+  padding-left: 5%;
 }
-
 .colored-bullets ul {
-    list-style: none; /* Remove default bullets */
+  list-style: none; /* Remove default bullets */
 }
 .colored-bullets ul li::before {
-    content: "\2022";  /* Add content: \2022 is the CSS Code/unicode for a bullet */
-    color: #FF4081; /* Change the color */
-    font-weight: bold; /* If you want it to be bold */
-    font-size: 28px;
-    display: inline-block; /* Needed to add space between the bullet and the text */
-    width: 1em; /* Also needed for space (tweak if needed) */
-    margin-left: -1.1em; /* Also needed for space (tweak if needed) */
+  content: "\2022"; /* Add content: \2022 is the CSS Code/unicode for a bullet */
+  color: #ff4081; /* Change the color */
+  font-weight: bold; /* If you want it to be bold */
+  font-size: 1.5em;
+  display: inline-block; /* Needed to add space between the bullet and the text */
+  width: 1em; /* Also needed for space (tweak if needed) */
+  margin-left: -1.1em; /* Also needed for space (tweak if needed) */
 }
-
-.auditionIMG{
-    padding-top: 10%;
-    height: 100%;
-    padding-left: 10%;
-    width: 100%;
+.auditionIMG {
+  padding-top: 10%;
+  height: 100%;
+  padding-left: 10%;
+  width: 100%;
+}
+.mobile {
+  display: none;
+}
+@media only screen and (min-width: 601px) {
+  .mobile {
+    display: none;
+  }
 }
 
+/* MOBILE DESIGNS*/
+@media only screen and (max-width: 600px) {
+  .mobile {
+    display: inline-block;
+  }
+  .title {
+    font-size: 5em;
+    text-align: center;
+  }
+  .colored-bullets ul li::before {
+    color: rgb(15, 185, 185); /* Change the color */
+  }
+  .dot {
+    display: none;
+  }
+  .dots {
+    display: none;
+  }
+  .audition-info {
+    margin-bottom: -20px;
+  }
+  .image2 {
+    margin-left: -8%;
+    width: 110%;
+  }
```

Comment:
In conclusion, just direct your attention here. This is most of the important stuff I ended up changing.